uploader.py

```diff
@@ -33,6 +30,12 @@
 	
 load_dotenv()
 
+OS = platform.system() 
+if OS in ['Linux', 'Java', 'Windows']:
+	MAC = False
+elif OS == "Darwin":
+	MAC = True
+
```

Comment:
Según las docs, si es que no logra identificar el OS va a retornar un string vacío, eso haría que no se inicializara la variable y se caería el código de abajo 😮 sería peluo que pase, pero mejor prevenir que lamentar

Reply:
Lo otro, quizás se podría usar algo tipo
```suggestion
OS = platform.system() 
if OS in ['Linux', 'Java', 'Windows']:
	key_command = Keys.CONTROL
elif OS == "Darwin":
	key_command = Keys.COMMAND
```
Así no teni que repetir código más abajo (no se me ocurrió un mejor nombre, tengo sueño y jet lag, lo siento)